qiskit/ignis/verification/randomized_benchmarking/dihedral.py

```diff
@@ -520,45 +592,194 @@ def from_circuit(self, circuit):
         append_circuit(elem, circuit)
         return elem
 
-    def compose(self, other):
+    def to_matrix(self):
+        """Convert operator to Numpy matrix."""
+        return self.to_operator().data
+
+    def to_operator(self):
+        """Convert to an Operator object."""
+        return Operator(self.to_instruction())
+
+    def compose(self, other, qargs=None, front=False):
         """Return the composed operator.
 
         Args:
             other (CNOTDihedral): an operator object.
+            qargs (None): None.
```

Comment:
Maybe mention that this is not yet implemented for this operator instead of just None.